source/routing/isochrone.cpp

```diff
@@ -140,27 +142,48 @@ static type::MultiPolygon merge_poly(const type::MultiPolygon& multi_poly, type:
     return multi_polys_merged;
 }
 
+struct InfoCircle {
+    type::GeographicalCoord center;
+    double distance;
+    int duration_left;
+    InfoCircle(const type::GeographicalCoord& center,
+           const double& distance,
+           const int& duration_left): center(center),
+                                      distance(distance),
+                                      duration_left(duration_left) {}
+};
+
+struct {
+  bool operator()(const InfoCircle& a, const InfoCircle& b) const {
+    return a.distance < b.distance;
+  }
+} furtherthan;
```

Comment:
That's a Cish syntax. You should use a CamelCase type, and declare the structure as the others:
```c++
struct TurtherThan {
   ...
};